lib/main.js

```diff
@@ -37,10 +37,12 @@ function run() {
             }
             yield exec.exec(`gprbuild -j0 -p -XSELFBUILD=False -P alr_env.gpr -cargs -fPIC`);
             core.addPath(path.join(process.cwd(), 'bin'));
-            // For some reason, this makes the action step to never finish on
-            // Windows
-            // console.log("Built version:");
-            // await exec.exec(`alr -n version`);
+            yield exec.exec(`alr -n toolchain --disable-assistant`);
```

Comment:
It seems odd to disable the assistant. What toolchain is going to be used?

Reply:
Whatever the user configures... The idea was to keep the actual situation, and people can install something with `alr toolchain --select`, or use system packages.

Otherwise I'm afraid we would break everybody's workflows (not that I know of any besides ours, which indeed broke in the devel branch)

Reply:
My opinion is that Alire toolchain support should be the default workflow, to make the simple cases as easy as possible.

Reply:
All right, I'll revert this change.

Reply:
What we could do instead is have a default to install the latest `gnat_native` and `gprbuild`:
`$ alr toolchain --select gnat_native gprbuild`
And have an option in the Action to select something different (and also an custom install dir):
```
    - uses: alire-project/setup-alire@latest-stable
      with: 
        toolchain: gnat_native=10.3.1 gprbuild
```
```
    - uses: alire-project/setup-alire@latest-stable
      with: 
        toolchain: gnat_native gprbuild
        toolchain_dir: ~/cache_dir
```
```
    - uses: alire-project/setup-alire@latest-stable
      with: 
        toolchain: --disable-assistant
```

Reply:
Right, more flexible